test/index.ts

```diff
@@ -578,7 +578,7 @@ test('signedTypeData_v4', (t) => {
     domain: {
         name: 'Ether Mail',
         version: '1',
-        chainId: 1,
+        chainId: 9007199254740992,
```

Comment:
Instead of changing this test can we add a new test? Otherwise the `try` part of the `try`/`catch` will never get hit.